src/openzaak/components/catalogi/api/serializers/informatieobjecttype.py

```diff
@@ -19,16 +16,42 @@
 )
 
 
-class OmschrijvingGeneriekSerializer(GegevensGroepSerializer):
```

Comment:
`GegevensGroepSerializer` is created to automate dealing with `GegevensGroepType` objects. In ZGW standards some properties sometimes are grouped into objects, but they have 1-1 relation with the resource and actually just a grouped collection of resource properties. For example `resultaattype.brondatum_archiefprocedure` -  https://www.gemmaonline.nl/index.php/Imztc_2.2/doc/gegevensgroeptype/resultaattype.brondatum_archiefprocedure

In the standards they are called `Gegevensgroeptype`, we keep the similar name for them. Since they are quire common in the standard we created special class and serializer to display them in RESTful API. 
One of our assumptions was that GegevensGroepType` should always be present. I didn't pay attention to it while implementing `InformatieObjectType.omschrijving_generiek`, which resulted in errors during the import. The usage of normal DRF serializer should fix it